.github/workflows/auto-update-versions.yml

```
@@ -0,0 +1,27 @@
+name: Auto-update versions
+
+on:
+  workflow_dispatch:
+  schedule:
+    # hourly at minute 10
+    - cron: "10 * * * *"
+
+jobs:
+  auto-update-versions:
+    name: Auto-update versions
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Use CLA approved github bot
+        run: |
+          git config user.name opentelemetrybot
+          git config user.email 107717825+opentelemetrybot@users.noreply.github.com
+
+      - name: Auto-update
+        run: |
+          ./.github/workflows/scripts/auto-update-version.sh "opentelemetry-collector-releases" "collectorVersion" "content/en/docs/collector/_index.md"
+          ./.github/workflows/scripts/auto-update-version.sh "opentelemetry-java" "javaVersion" "content/en/docs/instrumentation/java/_index.md" "content/en/docs/instrumentation/java/manual.md"
```

Following #2352, we can drop the second file:

```suggestion
          ./.github/workflows/scripts/auto-update-version.sh "opentelemetry-java" "javaVersion" "content/en/docs/instrumentation/java/_index.md"
```

---

⚠️ If `auto-update-version.sh` runs twice, won't the second run inherit the branch from the first run?

---

If so, the script can save the branch when it starts and change back to that branch when it's done.

---

fixed by resetting between each update